pages/education/apply-for-education-benefits/application/1990N.md

```diff
@@ -1,15 +0,0 @@
----
-title: Apply For Education Benefits
-heading: Apply for education benefits
-entryname: 1990n-edu-benefits
-layout: page-react.html
-description: Use your VA education benefits to pay for college or training programs. Find out which documents you’ll need to apply for benefits, and start your online application today.
----
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
-id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/">Home</a></li>
-    <li><a href="/education/">Education</a></li>
-    <li><a aria-current="page" href="/education/how-to-apply/">Apply for education benefits</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-b3ada11b8417a98561675ac96adb6b03R5.